docs/hybrid-slurm-cluster/inter-gcp-project-hybrid-slurm.md

```diff
@@ -0,0 +1,540 @@
+  * Set metadata in project B that inform the burst compute nodes how to
+    configure themselves.
+  * Create pubsub actions triggered by changes to the hybrid configuration.
+
+Either in the blueprint directly or on the command line, update the following
+deployment variables in the [hybrid-configuration.yaml] blueprint:
+
+* **_project\_id:_** The ID of project B.
+* **_static\_controller\_hostname:_** The fully qualified internal hostname of
+  the static cluster's controller in project A. The format is
+  `<<instance_name>>.c.<<Project_A_ID>>.internal`.
+
+If the deployment vars have been added directly to the blueprint, the following
+command will create the deployment directory:
+
+```shell
+ghpc create docs/hybrid-slurm-cluster/blueprints/hybrid-configuration.yaml
+```
+
+To create the deployment directory with deployment variables passed through the
+command line, run the following command with the updated values of
+`<<Project_B>>`, `<<Hostname>>` and `<<Homefs_IP>>` instead:
+
+```shell
+ghpc create docs/hybrid-slurm-cluster/blueprints/hybrid-configuration.yaml --vars project_id="<<Project_B>>",static_controller_hostname="<<Hostname>>.c.<<Project_A>>.internal"
+```
+
+If successful, this command will provide 3 terraform operations that can be
+performed to deploy the deployment directory. They should look similar to the
+following:
+
+```shell
+Terraform group was successfully created in directory peering-networks-demo/primary
+To deploy, run the following commands:
+  terraform -chdir=hybrid-config/primary init
+  terraform -chdir=hybrid-config/primary validate
+  terraform -chdir=hybrid-config/primary apply
+```
+
+Execute the terraform commands to create the hybrid configuration. A directory
+in `hybrid-configuration/primary` named `hyrid/` should be created which
+contains a `cloud.conf` file, `cloud_gres.conf` file and a set of support
+scripts.
+
+[hybridmodule]: ../../community/modules/scheduler/schedmd-slurm-gcp-v5-hybrid/README.md
+
+### Install and Configure Hybrid on the Controller Instance
+
+> **_NOTE:_** Many of the manual steps in this section have been adapted from the
+> hybrid documentation in [Slurm on GCP][slurm-gcp]. The source document can be
+> found at [docs/hybrid.md][slurm-gcp-hybrid]
+
+Now that the hybrid configuration directory has been created, it needs to be
+installed on the controller VM instance. First, tar the directory:
+
+```shell
+cd hybrid-config/primary
+tar czvf hybrid.tar.gz hybrid
+```
+
+Copy the `hybrid.tar.gz` file to the controller VM instance. This can be done
+in whichever way is easiest for you, `gcloud compute scp` is used here.
+
+```shell
+gcloud compute scp --project="<<Project_A>>" --zone=us-central1-c ./hybrid.tar.gz "<<Controller_Name>>:~"
+```
+
+Now SSH to the controller VM either using the console or the following gcloud
+command:
+
+```shell
+gcloud compute ssh --project="<<Project_A>>" --zone=us-central1-c "<<Controller_Name>>"
+```
+
+Decompress the `hybrid.tar.gz` file:
```

Comment:
Add a line telling user to ssh to controller

Reply:
There is a section just above telling the user to SSH to the controller with gcloud.